doc/doc-site/docs/off-chain/rust-sdk.md

```diff
@@ -1,3 +1,251 @@
+To do this, we use the helper function `econia_sdk::entry::register_market_base_coin_from_store`.
+
+Once that is done, we can use the result to call the `submit_tx` function.
+
+We then use an `EconiaViewClient` (that is dropped as soon as we finish using it) to get the market ID of the created market.
+
+Expected output:
+
+```
+==========Create a market for eAPT/eUSDC==========
+Market created with ID: 1
+Press enter to continue (next step: Set up account A)
+```
+
+The expected output might differ if you already ran the script once without resetting the chain.
+
+Now, the first step is finished.
+To go to the next step, press enter.
+You will have to do this after each step.
+
+### Step 2: Set up account A
+
+We will use the `account` helper function (you can find its definition in the script's source).
+The function initiates an account with some funds from the Aptos faucet.
+The function returns a tuple with the address of the newly created account and an `EconiaClient` corresponding to that account.
+In order to make a call with this account, you'll have to use this client.
+You will have to use multiple clients **and it is important not to mix them up**.
+
+Next, we'll fund the accounts with some example coins (eAPT and eUSDC).
+To do so, we use the `fund` helper function (you can find its definition in the script's source).
+
+Now that that's out of the way, we'll register the account to the market.
+This is an important step that all accounts have to do before participating in a market.
+For this, we'll use the `register_market_account` function that will create the appropriate `EntryFunction` for us.
+We then submit it.
+
+We'll deposit some coins to be used to open orders.
+For this, we use `deposit_from_coinstore`.
+
+You should see the following (`ADDR_A` would be a randomly generated address):
+
+```
+=================Set up account A=================
+Minted eAPT to ADDR_A
+Minted eUSDC to ADDR_A
+Registered market account for ADDR_A
+367
+Deposited eAPT from coinstore for account ADDR_A
+Deposited eUSDC from coinstore for account ADDR_A
+ADDR_A was successfully set up
+Press enter to continue (next step: Place two limit orders with account A)
+```
+
+Account A is now set up and ready to be used.
+
+### Step 3: Place two limit orders with account A
+
+As the title says, it's showtime.
+Let's place some orders.
+
+```
+======Place two limit orders with account A=======
+Placed bid order for account ADDR_A
+Placed ask order for account ADDR_A
+Press enter to continue (next step: Set up account B)
+```
+
+This is a bit more complex than previous code.
+Let's inspect the `place_limit_order_user_entry` function.
+Its first argument is the Econia package address, as is the case for all other functions in the `entry` module.
+Then comes two type arguments, base and quote.
+These are Aptos type arguments: see in the `init` function to review how they are created, or look into the Aptos docs for more details.
```

Comment:
How about a line break after the colon, and then take out "in" in "see in the `init`"?